models/cow_protocol/tx_hash_labels/early_investment/cow_protocol_tx_hash_labels_early_investment.sql

```diff
@@ -0,0 +1,8 @@
+{{
+    config(
+        alias='tx_hash_labels_early_investment',
+        post_hook='{{ expose_spells_hide_trino(\'["ethereum"]\', "sector", "tx_hash_labels", \'["gentrexha"]\') }}'
```

Comment:
Is hide trino really what we want here. Does this mean labels are not yet part of "v3" aka dune sql.

Reply:
I don't know about v3. But I continued hiding them since I didn't want to pollute our spells with the labels and have people finding those instead of our important ones.